### ISSUE TITLE
Turn the Upward Diagonal swing trajectory on the Pitchfork into a straight line.

### ISSUE THREAD

**Issue by maverickson** [Tsuey/L4D2-Community-Update#396]
### Suggestion

The shovel and the pitchfork share an animation where they swing from Southeast to Northwest of the screen. 

Every swing trajectory on the shovel, including the Southeast to Northwest animation, are all straight lines between East and West.

https://github.com/Tsuey/L4D2-Community-Update/assets/82732916/fdbe5017-13cf-4914-9118-2f5c23b09a67

The pitchfork has a specific trajectory for the Southeast to Northwest animation, as it's one of the few to have a directly diagonal trajectory like the bats.

https://github.com/Tsuey/L4D2-Community-Update/assets/82732916/84bcc8ff-b468-4b71-9fcf-a1391305d752

With the balance of the pitchfork deeming it a bad melee weapon due to its odd swing trajectories, evening out the trajectory of the swing to be a straight line from East to West will make it more consistent in its kill power, consistent with the shovel, and improve its utility in the game.

**Comment by alexiscoutinho** [comment]
I think your suggestion would make them less consistent. I would prefer the trajectories to match the animations, thus the shovel should have a diagonal trajectory for the diagonal swing animation. If making the pitchfork less of a meme weapon was desired, then I think the vertical animation shouldn't be the first. But surely there was a big discussion and consensus already in TLS update to add a shitty, albeit unique melee to the game.

**Comment by maverickson** [comment]
I think the pitchfork is quite unique and has a place, and I know for a fact that it wasn't meant to be a purposefully "shitty meme weapon" and I think calling it that is very callous. Where I would advocate for animation fixes, the CS:S knife proved that it's far better that we have straight cut trajectories regardless of the animation. As long as the animation sells the idea of where the blade is going, for gameplay and consistency sake, it should not be a literal translation of the movement. 

The shovel is in a really good spot right now, tweaking that _random swing animation_ to have the trajectory follow it would actually make the weapon worse. Again, the CS:S knife proves this.

**Comment by Tsuey** [comment]
There's a clear consensus that something about the pitchfork should be changed; exactly what should be changed is a different story, so all ideas are highly appreciated.

While not a meme weapon, it could have that reputation since it's meant to be unique and stand out from the others. It is inspired by the pitchfork from the game [Blood](https://www.youtube.com/watch?v=Qo6oOYI_00I). Before this issue was posted, @ltrockyy already had a slight edit in mind to raise the speeds of the weapon, you should be able to [download that vpk here](https://cdn.discordapp.com/attachments/717931797786132492/1136786131514568776/pitchfork_adjustment.vpk). It's not what you've asked, but it'd be cool if you could try it and give us continued feedback.

The other suggestion we get a lot -- Splinks himself even originally pitched it back in 2021 -- was moving the forward thrust attack to the 2nd or 3rd swing animation, since most players tend to use melee weapons to get out of tight binds and only do a single swing. So that approach would make the first swing a side hit.

Regardless of the final approach, maintaining the original concept from Blood would still be desired.

You mentioned the knife, we've been trying to get that [change](https://github.com/Tsuey/L4D2-Community-Update/blob/master/changelogs/Future_Update_01.md#weapons) in for a while and we actually expect to get that in the game soonish, so the pitchfork won't be quick and will need a lot of feedback / iteration.

**Comment by alexiscoutinho** [comment]
Well, another idea to keep the pitchfork's uniqueness, but to make it less memy would be to turn it into a niche weapon (good for only one thing). And the niche would be killing Witches, perhaps via a 2 or 3 hit kill on her (instead of 4).

**Comment by maverickson** [comment]
Just know that the only suggestion

**Comment by maverickson** [comment]
Sorry, misclicked.
I quite like the Pitchfork where it's at, and I respect its call to be as close to Blood as possible. It's why I'm only asking to go about the approach that I suggested, as it's the least changing yet still highly effective possible option in my opinion. No animation change, no model change, no animation reordering, just a change in one swing trajectory, as I believe that will fix a lot of the Pitchfork's functionality.

**Comment by Tsuey** [comment]
Yeah, I don't think we've heard this particular approach before, and in my opinion it's thinking outside the box and kept simple. The hard part always comes down to getting enough feedback to feel confident enough to make a change -- so any conversation, even if the linked VPK sounds bad, does significantly influence direction. It's very easy to get tunnel vision when feedback is limited to only a few devoted, and often more experienced, subset of the playerbase.

We started with the knife because even those who have gotten used to its TLS behavior, still acknowledge that it's not intuitive, and the sentiments seem uniquely unanimous compared to the other re-balancing / tweaks we'd like to try where we have 3-4 ideas in mind, like the pitchfork.
